llm/server.go

```diff
@@ -278,7 +279,7 @@ func NewLlamaServer(gpus discover.GpuInfoList, model string, ggml *GGML, adapter
 			continue
 		}
 
-		if strings.HasPrefix(servers[i], "cpu") {
+		if strings.HasPrefix(servers[i], "cpu") || (builtin && !(runtime.GOOS == "darwin" && runtime.GOARCH == "arm64")) {
```

Comment:
I'm ok with the `builtin` runner but wonder if we should actually label it the build it is – e.g. `cpu` on macOS amd64, `metal` on arm64 macOS etc. Not sure how big of a lift that would take here?